full-vnc/novnc-index.html

```diff
@@ -1 +1 @@
-<html><head><meta http-equiv="Refresh" content="0; url=vnc_lite.html?scale=true"></head></html>
+<html><head><meta http-equiv="Refresh" content="0; url=vnc.html?scale=true&autoconnect=true"></head></html>
```

Comment:
I wonder what the difference between `?scale=true` and `?resize=scale` is.

Reply:
true does not work on vnc.html on my end

Reply:
Fixed